### ISSUE TITLE
/p reset & /p dispose broken?

### ISSUE THREAD

**Issue by ShushImSam** [jasonwynn10/MyPlot#226]
- [x] This issue isn't duplicated - you can check if it is by using the search bar located at the top left hand corner and select "Issues" on the left.
- [x] This issue includes appropriate markdown for sections - e.g. code blocks for crash dumps.
- [x] This issue is understandable - feel free to use your native language to write issues if you are not comfortable with English.

## Issue description
What problem do you have?
/p reset & /p dispose does not work on the lastest builder

## Reproducing the issue
1. Generate a new plot world and try to dispose
2. ...

## Client information
PocketMine-MP Version: 3.9.3
Plugin Version: 1.6.2
PHP version: 7.3 (default)

## Optional information
```
Put a console error here, but please make sure you have your debug level set to 2 in your pocketmine.yml
```

**Comment by jasonwynn10** [comment]
What are you expecting to happen and what is actually happening?

**Comment by ShushImSam** [comment]
/p dispose says Plot Disposed but doesn't actually dispose it, you keep the Plot ownership

/p resets says it successfully reset and dispose , tps you out of the plot but stays your plot.

I expect the plot to be unclaimed

**Comment by jasonwynn10** [comment]
What data provider are you using?

**Comment by ShushImSam** [comment]
yaml

**Comment by jasonwynn10** [comment]
Unable to reproduce

**Comment by jasonwynn10** [comment]
Whoops. I spoke too soon. This is an issue with the cache system in the YAML and JSON providers.

**Comment by ShushImSam** [comment]
Thanks god. I was scratching my head right there :P

**Comment by jasonwynn10** [comment]
fixed in 196da829c6c28aab92cb075dd3f57035d686a4ef
